nise/generators/gcp/cloud_storage_generator.py

```diff
@@ -62,7 +62,12 @@ def _update_data(self, row):  # noqa: C901
 
         # All upper and lower bound values were estimated for each unit
         # Currently our only usage unit & pricing unit is bytes-seconds & gibibyte month
-        amount = self.fake.pyint(min_value=1000, max_value=100000)
+        if self.attributes:
+            if self.attributes.get("usage.amount"):
+                amount = self.attributes.get("usage.amount")
+                amount_defined = True
+        if not amount_defined:
```

Comment:
So, this variable would never be defined unless you use usage.amount in the yaml. Also, I think I may know of some ways to simply this too. Let me know if you want me to go over them with you or just push up a change.

Reply:
yeah, if you wanna push up a change that works for me